crowdfunding/projects/serializers.py

```diff
@@ -0,0 +1,63 @@
+from rest_framework import serializers
+
+from .models import Project, Pledge
+from users.serializers import CustomUserSerializer
+
+class PledgeSerializer(serializers.ModelSerializer):
+    supporter = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Pledge
+        fields = ['id', 'amount', 'comment', 'anonymous', 'project', 'supporter']
+        read_only_fields = ['id', 'supporter']
+
+    def get_supporter(self, instance):
+        
+        if instance.anonymous:
+            return "The Lego Elves!"
```

Comment:
cute anonymous, however, i would have probably returned `None` and then let the front end do the joke 😄